packages/odyssey-react-mui/src/index.ts

```diff
@@ -70,6 +70,7 @@ export * from "./createShadowDom";
 export * from "./createUniqueId";
 export * from "./Dialog";
 export * from "./Fieldset";
+export * from "./FieldComponentProps";
```

Comment:
I don't think this needs to be exported. It's internal-only.

Reply:
I exported it to use in Storybook. But, could change my import to this if we prefer
```js
import { fieldComponentPropsStorybookMetaData } from "@okta/odyssey-react-mui/FieldComponentProps";
```